docs/tutorials/gettingstartedexample/gse-create-tdms.md

```diff
@@ -0,0 +1,193 @@
+- Line Start following a Line End has Max Delay
+- Frame End following Line End has Min Delay
+    
+The rules above can be summarized to:
+
+| Current Packet | Next Packet | Delay (cycles) between end of Current and start of Next Packet |
+|-|-|-|
+| Frame Start | Line Start | Min Delay |
+| Frame Start | Long Packet | Min Delay |
+| Frame End | Frame Start | Determined by desired frame rate, but must at least equal Min Delay |
+| Line Start | Long Packet | Min Delay |
+| Line End | Line Start | Max Delay |
+| Line End | Frame End | Min Delay |
+| Long Packet | Long Packet | Max Delay |
+| Long Packet | Line End | Min Delay |
+| Long Packet | Frame End | Min Delay |
+    
+    
+The figure below shows what the timing of the generated LLP packets will look like with **include line sync packets** set to false.
+   ![No Line Sync Packets Diagram](../../images/PXIe-148X-CreateTDMS-NoLineSyncPackets-Diagram.png)
+
+The figure below shows what the timing of the generated LLP packets will look like with **include line sync packets** set to true.
+   ![Line Sync Packets Diagram](../../images/PXIe-148X-CreateTDMS-LineSyncPackets-Diagram.png)
+    
+    
+The following scenarios will illustrate the above concepts in more detail.
+        
+## Determining the maximum frame rate for a given frame size    
+> Note: For the purposes of this tutorial, all input control values not specified should be left as the default value.
+    
+1. Double click the Create CSI-2 Packet TDMS Files VI in the LabVIEW project.
+
+2.  Set the **Number of Frames** control to 1. Setting this to 1 will let you quickly iterate over configuration settings and getting feedback with the **Actual Frame Rate (fps)** indicator.
+    
+3. Set the **desired frame rate (fps)** to 1000. This will make the frame period much smaller than the interpacket delay and LLP packet duration.
+    
+4. Run the VI and look at the **Actual Frame Rate (fps)** indicator. This tells you for a given **Frame Data Configuration** and **Frame Timing Configuration** what the maximum achieveable FPS is.
+    
+5. Now the set the **desired frame rate (fps) to 40. This will make the frame period larger than the framme perod required for the interpacket delay and LLP packet duration.
+    
+6. Run the VI and look at the **Actual Frame Rate (fps)** indicator. It shows 40 fps.
+    
+7. Double click the TDMS File Viewer VI in the LabVIEW project.
+    
+    ![Open TDMS File Viewer VI](../../images/PXIe-148X-GSE-TDMSFileViewer.png)
+
+8. Select the correct location for the **TDMS File Directory** control. By default, TDMS files generated were saved to <font face = "courier new">\<yourprojectdir\>\\Host\\Gen\\TDMS Files</font>.
+
+    > Note: The directory browse window will not show the TDMS files located in the "TDMS Files" directory.
+    
+9. Run the TDMS File Viewer VI and verify you have one frame of data. You should have a Frame Start packet, followed by 1080 RAW12 packets, followed by a Frame End packet.
+    
```

Comment:
Consider adding a step 10 that directs them to change the frame size, observe the altered frame rate, and then repeat using the viewer and tell them what to expect.  I think this could be a condensed single step, something like:

Repeat steps 5 through 9 with a frame size of X*Y and observe that you should see a frame rate of Z and you should see in the TDMS File Viewer VI Y RAW12 packets, etc, etc